docs/advanced.rst

```diff
@@ -1589,24 +1589,56 @@ It is also possible to call python functions via ``operator()``.
     py::object result_py = f(1234, "hello", some_instance);
```

Comment:
This sounds good to me.

Sent from my iPhone

> On 05 Sep 2016, at 18:06, Dean Moldovan <notifications@github.com> wrote:
> 
> In docs/advanced.rst:
> 
> > @@ -1589,24 +1589,56 @@ It is also possible to call python functions via ``operator()``.
> >      py::object result_py = f(1234, "hello", some_instance);
> What do you think about creating a new page dedicated to the Python C++ API? The wrapper classes (str, tuple, dict) are also a bit underdocumented, but the the Advanced page is already getting quite long and hard to navigate. (It's probably worth splitting of additional sections which form a logical unit (like buffer/numpy/eigen) but that's a topic for a different PR.)
> 
> ―
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 
> {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/pybind/pybind11","title":"pybind/pybind11","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/pybind/pybind11"}},"updates":{"snippets":[{"icon":"PERSON","message":"@dean0x7d in #372: What do you think about creating a new page dedicated to the Python C++ API? The wrapper classes (`str`, `tuple`, `dict`) are also a bit underdocumented, but the the Advanced page is already getting quite long and hard to navigate. (It's probably worth splitting of additional sections which form a logical unit (like buffer/numpy/eigen) but that's a topic for a different PR.)"}],"action":{"name":"View Pull Request","url":"https://github.com/pybind/pybind11/pull/372/files/1f13b0c69e27b520b319fdcc170e0e52fb957644#r77490851"}}}